platform/darwin/src/MGLFeature.mm

```diff
@@ -457,7 +457,14 @@ static CLLocationCoordinate2D toLocationCoordinate2D(const mbgl::Point<T> &point
         NSExpression *identifierExpression = [NSExpression expressionForConstantValue:identifier];
         feature.id = [identifierExpression mgl_featureIdentifier];
     }
-    feature.properties = [attributes mgl_propertyMap];
+
+    // Expression 'feature.properties = [attributes mgl_propertyMap]' for attributes dictionary of nil value
+    // prouces std::unordered_map with max_load_factor of 0.0, while the default value must be 1.0.
+    // Therefore, overflow exception is thrown when new key is inserted into the mbgl::Feature properties.
+    if (attributes) {
```

Comment:
If `attributes` can be `nil`, then it should be marked as `nullable` in the signature of this function.

Reply:
identifier can be nullable too, should it be also fixed?

Reply:
👍 defined attributes as nullable